### PR TITLE
Fix showing age for ancient characters

### PR DESCRIPTION
**Pull request by robmart**
## Developer changelog:
- Should fix it so that you can't see a character's age if if they were born prior to year 20, aka 562 years or older at the first startdate
- The parts that were changed is:
    - Death events
    - Character tooltip
   
## How to test
- Make sure that ancient characters (age ???) don't have their actual age shown anywhere

# Todo:
- Replace all strings
    - Loc
        - [x] GetAge
        - [x] GetBirthYear
        - [x] GetBirthdate
        - [x] GetAgeInfo
    - GUI
        - [x] GetAge
        - [x] GetBirthYear
        - [x] GetBirthdate
        - [x] GetAgeInfo

### CODE REVIEW

File: common/customizable_localization/wc_general_custom_loc.txt [hunk 1]
```diff
@@ -267,4 +267,22 @@ GetCustomUnionName = {
 		fallback = yes
```
Comment (zumbak04) [inline common/customizable_localization/wc_general_custom_loc.txt:267]: You should replace all localization lines that mention `GetAge`, `GetBirthYear`, `GetBirthdate`, `GetAgeInfo`.
<details>
<summary>Click to expand</summary>

image
image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/customizable_localization/wc_general_custom_loc.txt (hunk 1 above)
Comment (robmart) [reply common/customizable_localization/wc_general_custom_loc.txt:267]: Oh god lord. Back to the code mines I go.

Comment (robmart) [reply common/customizable_localization/wc_general_custom_loc.txt:267]: Done

File: localization/replace/english/dlc/fp1/dlc_fp1_custom_loc_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿l_english:
```
Comment (arithon) [inline localization/replace/english/dlc/fp1/dlc_fp1_custom_loc_l_english.yml:1]: there's already a file called like that directly in the replace/english folder, is this one really necessary?

Comment (robmart) [reply localization/replace/english/dlc/fp1/dlc_fp1_custom_loc_l_english.yml:1]: Must have missed that one

File: localization/replace/english/bookmark/wc_bookmark_l_english.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿l_english:
+ BOOKMARK_CHARACTER_AGE:0 "Age: [BookmarkCharacter.GetAge|V]"
+ BOOKMARK_CHARACTER_INFO:2 "#high;bold [BookmarkCharacter.GetName]#!, #medium [BookmarkCharacter.GetAge]#!"
+ BOOKMARK_CHARACTER_AGE_DATA:0 "Age: [BOOKMARK_CHARACTER.GetAge]"
+ #What TODO?
```
Comment (zumbak04) [inline localization/replace/english/bookmark/wc_bookmark_l_english.yml:5]: What is a problem?

File: localization/replace/english/gui/wc_ruler_designer_l_english.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+﻿l_english:
+ RULER_DESIGNER_AGE:1 "#high [RulerDesignerWindow.GetAge]#! [Select_CString( GreaterThan_int32( RulerDesignerWindow.GetAge, '(int32)1' ), 'Years', 'Year' ) )]"
+ RULER_DESIGNER_BIRTH_YEAR:0 "Born: [RulerDesignerWindow.GetBirthYear] AD"
+ #What TODO?
```
Comment (zumbak04) [inline localization/replace/english/gui/wc_ruler_designer_l_english.yml:4]: Nothing. It's the Ruler Designer.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by arithon — APPROVED** [review]
loc LGTM

**Review by hamjustine — APPROVED** [review]

**Pull request merged**